ddtrace/bootstrap/sitecustomize.py

```diff
@@ -15,6 +15,10 @@
     '[dd.trace_id=%(dd.trace_id)s dd.span_id=%(dd.span_id)s] ' if logs_injection else ''
 )
 
+if logs_injection:
+    # immediately patch logging if trace id injected
+    from ddtrace import patch_all; patch_all(**dict(logging=True)) # noqa
```

Comment:
You probably want to explicitly only patch `logging`. This will patching everything **and** logging

```python
from ddtrace import patch; patch(logging=True)
```

Reply:
👍